### PR TITLE
HLD - Management support for Error Handling Framework

### PR DESCRIPTION
**Pull request by bhaveshdell**

### CODE REVIEW

File: doc/error-handling/SONiC_Mgmt_Error_Framework_HLD.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+## 3.4 SyncD
+Describe changes to syncd if applicable.
+
+## 3.5 SAI
+Describe [new/existing] SAI APIs used by this feature.
+
+## 3.6 User Interface
+### 3.6.1 Data Models
+
+A new sonic yang (sonic-error.yang) provides the interface for configuration and status.
```
Comment (kkvedula) [inline doc/error-handling/SONiC_Mgmt_Error_Framework_HLD.md:110]: Please fix the markdown formatting.

File: doc/error-handling/SONiC_Mgmt_Error_Framework_HLD.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+```
+module: sonic-error
+    +--rw sonic-error
+       +--rw ERROR_NEIGH_TABLE
+       |  +--rw ERROR_NEIGH_TABLE_LIST* [ifname prefix]
```
Comment (sivamukka) [inline doc/error-handling/SONiC_Mgmt_Error_Framework_HLD.md:115]: Key in the neighbor table is interface and IP address. Please replace prefix with IP address

File: doc/error-handling/SONiC_Mgmt_Error_Framework_HLD.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+Operation  :  create
+------------------------------
+Route      : 31.20.20.0/24
+Operation  :  create
+------------------------------
+------------------------------
+ERROR_NEIGH_TABLE
+------------------------------
+Interface  : Ethernet36
+Prefix     : 2.2.2.2/24
```
Comment (sivamukka) [inline doc/error-handling/SONiC_Mgmt_Error_Framework_HLD.md:165]: Replace prefix with IP address here

**Comment by sameerdell** [comment]
Closing incomplete feature.

**Pull request closed without merging**